transformation-engine/latest/install/msi.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Installation
+---
+
+The standalone Alfresco Document Transformation Engine is installed by using either an `.msi` or by installing the Document Transformation Engine T-Engine with the Transform Service, using Docker Compose.
+
+In previous versions the installation files were contained within a `.zip` file. This file also contained `.amp` files that enabled you to install the Document Transformation client into Alfresco Content Services. In the current version this is not possible.
+
+* [Install with MSI](#install-with-msi)
+* [Install T-Engine using Docker Compose](#install-t-engine-using-docker-compose)
+
+## Install with MSI
+
+> **Note:** When upgrading the Document Transformation Engine, the previous installation must be uninstalled first. If your old version of the Document Transformation Engine is earlier than 1.3.1, use the Control Panel **Uninstall a program** option to remove the old version, and then manually remove the Document Transformation Engine directory. By default, the Document Transformation Engine directory is `C:\Program Files (x86)\Transformation Engine\`. If your old version of the Document Transformation Engine is 1.3.1 or later, the new Document Transformation Engine MSI prompts you to uninstall the previous version. When the uninstall is complete, you can run the MSI package again to install the new version. There is no need to manually remove anything.
+
+1. Download `alfresco-document-transformation-engine-server-2.3.0.msi` from the [Alfresco Support Portal](http://support.alfresco.com){:target="_blank"}.
+
+2. Log into the Microsoft Windows Server as an administrator.
+
+3. Double click the `.msi` installer package, and then click **Next**.
+
+4. Review the supported software requirements, and then click **Next**.
+
+5. (Optional) Select DTE T-Engine.
+
+    > **Important:** If you do not have a T-Engine installed with Content Services, you must select this option. This allows transformation of MSG and EML files into PDF format using the Transform Service. If you already have a T-Engine installed with Content Services you should not select this option because it will produce an error.
```

Comment:
I'm not sure I understand this "Important" note. Since DTE 2.3 is ACS 7.0-only, it MUST have the t-engine wrapper installed. You can choose to have it installed onto the same server as the DTE Standalone Server by selecting this option in the msi installer. Or you can use the Docker image "quay.io/alfresco/transform-dte-engine" to provide the t-engine wrapper. While we could document how to move the t-engine the msi installs to a different server, I think that just unnecessarily complicates things. In any case, the t-engine (either installed by msi, or via Docker image) is required.

All a long winded way to say we should replace "If you do not have a T-Engine installed with Content Services," with "If you do not plan to use the DTE t-engine Docker image," in the first sentence.

The last two sentences should be removed. The second sentence seems to be from the Outlook t-engine doc and is misleading and not relevant. The third sentence is also misleading because again, we provide no mechanism to install the DTE t-engine into ACS, only onto the DTE standalone server or via Docker image.